core/consensus/babe/impl/block_executor.cpp

```diff
@@ -67,15 +76,21 @@ namespace kagome::consensus {
     auto new_block_hash =
         hasher_->blake2b_256(scale::encode(new_header).value());
     BOOST_ASSERT(new_header.number >= last_number);
-    return requestBlocks(last_hash, new_block_hash, std::move(next));
+    auto [_, babe_header] = getBabeDigests(new_header).value();
+    return requestBlocks(last_hash,
```

Comment:
Why return if it's void and is the end of function?

Reply:
It's lifehack for combine last fuction call and return. =)